datajunction-server/datajunction_server/api/nodes.py

```diff
@@ -472,6 +474,90 @@ def activate_a_node(name: str, *, session: Session = Depends(get_session)):
     )
 
 
+@router.delete(
+    "/nodes/{node_name}/hard/",
+    response_model=List[MaterializationConfigInfoUnified],
+)
+def hard_delete_a_node(
+    node_name: str,
+    *,
```

Comment:
What is this wildcard for?

Reply:
It just splits the positional and named arguments. So `hard_delete_a_node("foo", session)` won't work, you'd have to explicitly use `hard_delete_a_node("foo", session=session)`. I use it as more of a visual split to put all of the automatic dependency injected stuff after it, but I can remove it.

Reply:
Ah... visual split. Didn't realize. I don't mind seeing it there. Up to you.